### PR TITLE
Fix a typo error

### PR DESCRIPTION
**Pull request by AndrewTCEC**
Not sure if this error is consistent to every language, have a look. You can also change the primary lang spelling, so that all the translations will align automatically.

### CODE REVIEW

File: translate/rus.json [hunk 1]
```diff
@@ -162,7 +162,7 @@
 "exposure":"экспозиция",
 "extra options":"дополнительные настройки",
 "extra":"дополнительно",
-"fifty-move rule":"правило 50 ходов",
+"fifty-moves rule":"правило 50 ходов",
```
Comment (octopoulos) [inline translate/rus.json:165]: It's not a typo, it's fifty moves, but a fifty-move rule.
It's correct.

Comment (AndrewTCEC) [reply translate/rus.json:165]: Then I am not sure if this is correct....
Screenshot_20210303_003755_com android chrome

Comment (octopoulos) [reply translate/rus.json:165]: Yeah, this one is then incorrect. Must be fixed in cutechess cli. We'll have a look.

Comment (octopoulos) [reply translate/rus.json:165]: Actually I just added code to the GUI that corrects the mistake, so we don't need to modify cutechess-cli.

**Pull request closed without merging**